.github/workflows/lint.yml

```
@@ -1,7 +1,7 @@
 name: Lint Code Base
 
 on:
-  push:
+  pull_reqeust:
```

```suggestion
  pull_request:
```
a small typo correction

---

Thanks @katie-simmons! 👍